m_etcd/coordinator.go

```diff
@@ -319,7 +319,7 @@ startWatch:
 func (ec *EtcdCoordinator) parseTask(resp *etcd.Response) (task string, ok bool) {
 	// Sanity check / test path invariant
 	if !strings.HasPrefix(resp.Node.Key, ec.taskPath) {
-		metafora.Errorf("Received task from outside task path: %s", resp.Node.Key)
+		metafora.Errorf("%s received task from outside task path: %s", ec.name, resp.Node.Key)
```

Comment:
I know we talked about it, but would it be worth just moving to a context interface (maybe just `String()`) for tracing all Metafora logging?